app/assets/stylesheets/wpcc/section/_contributions.scss

```diff
@@ -9,8 +9,9 @@
   }
 }
 
-.contributions__helper {
+.popup-tip__container.contributions__helper {
   top: 0;
+  z-index: 2;
```

Comment:
We should make style changes to this element in Dough since this is a generic component and as far as possible it would be better to avoid making such changes in specific tools.

Reply:
@davidtrussler I've updated the css to be inline with how it's set on Step 1.

Reply:
Not sure if this can be set on Dough as it may affect other pages, if set on Dough maybe with a higher z-index?

Reply:
@margOnline Yes, I see that these styles already exist on this tool but they shouldn't. As far as possible these should be in the Dough component otherwise we will start overriding styles on the same element unnecessarily.